### ISSUE TITLE
Focus outline insufficient

### ISSUE THREAD

**Issue by amberhinds** [WordPress/twentytwentyfour#744]
https://github.com/WordPress/twentytwentyfour/blob/df92472089ede6fae5924c124a93c843b84e8cbd/theme.json#L800

The current focus outline is 1px dotted. This would be hard for people with low-vision to see and is a failure of WCAG 2.4.7:
Focus Visible (Level AA). 

The focus outline needs to be 2px solid.

**Comment by carolinan** [comment]
Moved to https://core.trac.wordpress.org/ticket/60334
